templates/master/00-master/openstack/files/openstack-mdns-publisher.yaml

```diff
@@ -0,0 +1,90 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/kubernetes/manifests/mdns-publisher.yaml"
+contents:
+  inline: |
+    kind: Pod
+    apiVersion: v1
+    metadata:
+      name: mdns-publisher
+      namespace: kube-system
+      creationTimestamp:
+      deletionGracePeriodSeconds: 65
+      labels:
+        app: kni-infra-mdns
+    spec:
+      volumes:
+      - name: resource-dir
+        hostPath:
+          path: "/etc/kubernetes/static-pod-resources/mdns"
+      - name: kubeconfig
+        hostPath:
+          path: "/etc/kubernetes/kubeconfig"
+      - name: conf-dir
+        hostPath:
+          path: "/etc/kubernetes/static-pod-resources/mdns"
+      initContainers:
+      - name: clusterrc-generation
+        image: quay.io/openshift-metalkube/kubeconfig-extractor:latest
```

Comment:
OK so I was wrong I see we are pulling in other containers here, so that's good.  However they will also need to be added to the release image.  See [the image-references file](https://github.com/openshift/machine-config-operator/blob/master/install/image-references).